### PR TITLE
Add chart component 📈

### PR DESCRIPTION
**Pull request by FGRibreau**
**New chart component**:

```hbs
{{> chart width='534' height='200' params='cht=lc&chd=s:cEAELFJHHHKUju9uuXUc&chco=76A4FB&chls=2.0,0.0,0.0&chxt=x,y&chxl=0:|0|1|2|3|4|5|1:|0|50|100&chg=20,50' alt='beautiful chart' }}
```

### CODE REVIEW

**Comment by FGRibreau** [comment]
Hum, since the chart is a picture, I don't think it will be possible the only way we have is to set width:100% without height, so the picture will resize depending on the container.

Would you like me to change the template so `height & width` will only be used for image chart size but not in the `` itself?

**Comment by leemunroe** [comment]
@FGRibreau I think so. Here are the Litmus tests with `height` attribute removed https://litmus.com/checklist/emails/public/8592bc5

**Comment by FGRibreau** [comment]
@leemunroe thanks for your feedback! I updated the PR :)

File: src/partials/components/chart.hbs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<table class="btn btn-{{ type }}" cellpadding="0" cellspacing="0">
```
Comment (leemunroe) [inline src/partials/components/chart.hbs]: Sorry just noticed this. Is there a reason we're using a btn class here? Assuming it was copied over from another component file?

I think for the purposes of the chart, we can change this to be just one table and give it a different class e.g.

```
<table class="chart" cellpadding="0" cellspacing="0">
  <tr>
   <td align="{{ align }}">
      <img width="{{{ width }}}" src="https://image-charts.com/chart?chs={{{ width }}}x{{{ height }}}&amp;{{{ params }}}" alt="{{{ alt }}}" />
   </td>
  </tr>
</table>

Thoughts?

**Comment by leemunroe** [comment]
🎉

**Pull request merged**